godot-core/src/builtin/color.rs

```diff
@@ -4,30 +4,511 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
  */
 
+use crate::builtin::inner::InnerColor;
+use crate::builtin::GodotString;
 use godot_ffi as sys;
+use std::ops;
 use sys::{ffi_methods, GodotFfi};
 
+/// Color built-in type, in floating-point RGBA format.
+///
+/// Channel values are _typically_ in the range of 0 to 1, but this is not a requirement, and
+/// values outside this range are explicitly allowed for e.g. High Dynamic Range (HDR).
 #[repr(C)]
-#[derive(Copy, Clone, Debug, PartialEq)]
+#[derive(Copy, Clone, Debug, PartialEq, PartialOrd)]
```

Comment:
Do you know if the ordering relation is also component-wise (r, g, b, a) in Godot?

Reply:
It is.